test_pkg/backend_for_get.go

```diff
@@ -0,0 +1,49 @@
+package test_pkg
```

Comment:
Thoughts on standardizing on a mock generator like [mockery](https://github.com/vektra/mockery)?

It'll generate the mock for backend based on the interface.
```
cd backend
mockery -name Backend
```

Reply:
Yep. I would love to do that. I just didnt want to propose new tool while we are close to release :-)